src/PJ_merc.c

```diff
@@ -48,11 +49,36 @@ static LP s_inverse (XY xy, PJ *P) {           /* Spheroidal, inverse */
     return lp;
 }
 
+/* Cf http://desktop.arcgis.com/en/arcmap/10.3/guide-books/map-projections/mercator.htm */
+#define AUX_SPHERE_TYPE_SEMI_MAJOR 0
```

Comment:
Why not use a enum here? You could use a switch below instead of the else if's. I think that is cleaner.

Reply:
Done